nodes/parachain/Cargo.toml

```diff
@@ -74,6 +74,7 @@ cumulus-client-consensus-proposer.workspace = true
 cumulus-client-service.workspace = true
 cumulus-primitives-core.workspace = true
 cumulus-relay-chain-interface.workspace = true
+itertools = "0.10.5"
```

Comment:
Let's use the workspace https://github.com/Polimec/polimec-node/blob/205b7c41dd7c7f5ba9927694397e49d5bcf10f91/Cargo.toml#L88